misc/config_tools/board_inspector/legacy/board_parser.py

```diff
@@ -46,6 +46,16 @@ def check_env():
     if os.path.exists(PY_CACHE):
         shutil.rmtree(PY_CACHE)
 
+    # check msr driver
+    res = subprocess.Popen("lsmod | grep -i ^msr",
+                            shell=True, stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE, close_fds=True)
+    line = res.stdout.readline().decode('ascii')
+    if not line:
+        subprocess.Popen("sudo modprobe msr",
+                            shell=True, stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE, close_fds=True)
```

Comment:
Since `modprobe` can fail as well, we should check the existence of `/dev/cpu/X/msr` again and raise a fatal error if it does not exist even after probing the `msr` module. Check the status of the `modprobe` command may also do the trick.

BTW, we do not need `sudo` here as the script has already verified itself to be executed by the root user.